packages/pieces/community/google-calendar/src/lib/actions/get-events.ts

```diff
@@ -63,11 +69,14 @@ export const getEvents = createAction({
       end_date,
       search,
       event_types,
+      singleEvents,
     } = configValue.propsValue;
     const { access_token: token } = configValue.auth;
     const queryParams: Record<string, string> = { showDeleted: 'false' };
     let url = `${googleCalendarCommon.baseUrl}/calendars/${calendarId}/events`;
 
+    queryParams['singleEvents'] = singleEvents ? 'true' : 'false';
```

Comment:
Can you please check if singleEvents is not null then set this parameter, in that way we can achieve backward compatibility and then please change the version to 0.5.6

Reply:
Sure, I will plan some time to work on this today.

Thanks for your review.

Reply:
I'm not sure this null verification was required because the default value is false. Any how. I've made it explicit.

Change done.